ur10_moveit_config/config/ompl_planning.yaml

```diff
@@ -34,7 +34,6 @@ manipulator:
     - TRRTkConfigDefault
     - PRMkConfigDefault
     - PRMstarkConfigDefault
-  projection_evaluator: joints(shoulder_pan_joint,shoulder_lift_joint)
```

Comment:
Is there a way to configure a specific planner?

If I understood you correctly, removing the projection setting leads MoveIt to choose something other than `LBKPIECE1` which is good, but if the selection behaviour changes, we could run into issues again.